mta-multi-tenant.yaml

```diff
@@ -48,10 +48,15 @@ modules:
   - name: bookshop-mt-sidecar
 # ------------------------------------------------------------
     type: nodejs
-    path: mtx/sidecar/gen
+    path: mtx/sidecar
     parameters:
       memory: 256M
       disk-quota: 512M
+    build-parameters:
+      builder: custom
+      build-result: gen
+      commands:
+        - npm run build
```

Comment:
As we have this build step, do we need the pre-build step in the line 11 (- npm install --prefix mtx/sidecar --production)? This would be really good if we could get rid of it, as it works only properly on Mac.

Reply:
Good catch! It seems that it is not needed anymore.